dist/common/scripts/scylla_util.py

```diff
@@ -380,9 +380,9 @@ def __init__(self):
 
     @staticmethod
     def is_azure_instance():
-        """Check if it's Azure instance via DNS lookup to metadata server."""
+        """Check if it's Azure instance via DNS lookup to IMDS server."""
         try:
-            addrlist = socket.getaddrinfo('metadata.azure.internal', 80)
```

Comment:
What's wrong with that? Please add the explanation to the patch changelog.

Reply:
@tarzanek the url is probably missing from /etc/hosts file. we had the same discusstion in https://github.com/scylladb/scylla/pull/8596

Reply:
but your change is merged @yaronkaikov, so how come this is still text and not octets?

Reply:
ah, you reverted, ok, I see now, well ... so is this Azure bug then? but if Felipe hit it and you hit it Yaron, then we should likely switch to IP

Reply:
unless you know a way to repro and we can report to Azure, but despite that we should work in both cases

Reply:
I couldn't find any references to metadata.azure.internal. Not sure where this came out from. 

Azure IMDS docs has no mentions to `metadata.azure.internal`. Wasn't this a simply copy'n'paste from GCP class? :-) 

I'll update the description for clarity.